### DISCUSSION TITLE
Would it be possible to add the "Go to..." option in the preview window?

### DISCUSSION THREAD

**Discussion by Blu3train** [buliasz/tesstrain-windows-gui#28] [Ideas]
I'm using the preview window to look for any errors in the text files by scrolling through the samples with the "Find next Wrong OCR" button and checking when it stops if the text matches.

I find it very convenient, but if I have a lot of files and for some reason I stop checking, when I resume it would be convenient to be able to enter the sample number to go to.

For example, I am checking a training with about 5000 images and I stopped to check about 2000 images. Now resume and get to 2000 to continue, it is very uncomfortable.

**Comment by Blu3train** [comment]
I did a PR

**Comment by buliasz** [comment]
Thanks, I've merged and updated a bit.
